APE_coordinated_pkg.rst

```diff
@@ -0,0 +1,235 @@
+An Updated Model for the Affiliated Package Ecosystem
+-----------------------------------------------------
+
+author: Erik Tollerud, 2017 Astropy Coordination Meeting Attendees
+
+date-created: 2018 February 22
+
+date-last-revised: 2018 February 22
+
+type: Process
+
+status: Discussion
+
+
+Abstract
+--------
+
+A revision of the concept of "affiliated packages" is proposed.  The idea of
```

Comment:
Here and elsewhere, I think this should say 'Astropy-affiliated packages' to be specific. We often say 'affiliated packages' colloquially, but the correct/formal term includes 'Astropy'. Or maybe after the first time you can say (hereafter 'affiliated packages')

Reply:
Good point, but I think I'll choose the second way just to lower the clutter.